config/LOCAL.yaml

```diff
@@ -0,0 +1,30 @@
+# Control parameters
+
+#Thrusters
+/proc_control/mpc/tmax : 40.0
+/proc_control/mpc/tmin : -30.0
+
+#TargetReached
+/proc_control/target_reached/linear_tolerance : 0.2 # meters
+/proc_control/target_reached/angular_tolerance : 0.1 # radians
+/proc_control/target_reached/time_in_tolerance : 3 # seconds
+
+#MPC Gains Default
+/proc_control/mpc/gains/default/ov : '70,60,70,90,90,90,90,0,0,0,0,0,0'
+/proc_control/mpc/gains/default/mv : '0.2,0.2,0.2,0.2,0.2,0.2,0.2,0.2'
+/proc_control/mpc/gains/default/mvr : '0.1,0.1,0.1,0.1,0.3,0.3,0.3,0.3'
+
+#MPC Gain Mode 10
+/proc_control/mpc/gains/c10/ov : '30,30,30,45,45,45,45,0,0,0,0,0,0'
+/proc_control/mpc/gains/c10/mv : '0.2,0.2,0.2,0.2,0.2,0.2,0.2,0.2'
+/proc_control/mpc/gains/c10/mvr : '0.4,0.4,0.4,0.4,0.5,0.5,0.5,0.5'
+
+#MPC Gain Mode 11
+/proc_control/mpc/gains/c11/ov : '30,30,30,45,45,45,45,0,0,0,0,0,0' 
+/proc_control/mpc/gains/c11/mv : '0.2,0.2,0.2,0.2,0.2,0.2,0.2,0.2'
+/proc_control/mpc/gains/c11/mvr : '0.4,0.4,0.4,0.4,0.5,0.5,0.5,0.5'
+
+#MPC Gain Mode 19
+/proc_control/mpc/gains/c19/ov : '0,0,0,0,0,0,0,20,20,20,20,20,20'
+/proc_control/mpc/gains/c19/mv : '0.2,0.2,0.2,0.2,0.2,0.2,0.2,0.2'
+/proc_control/mpc/gains/c19/mvr : '0.1,0.1,0.1,0.1,0.3,0.3,0.3,0.3'
```

Comment:
Maybe we could have two LOCAL files, one for each sub ?

Reply:
Technically, the same config should be used locally as in the pool. Local Config is more a debug feature to leave the config that works in the pool intact